src/widgets/capture/capturewidget.cpp

```diff
@@ -129,15 +131,17 @@ CaptureWidget::CaptureWidget(uint id,
         move(currentScreen->geometry().x(), currentScreen->geometry().y());
         resize(currentScreen->size());
 #else
-        // Comment For CaptureWidget Debugging under Linux
+
+#if (defined(QT_NO_DEBUG) && defined(Q_OS_LINUX))
```

Comment:
Why is behind QT_NO_DEBUG?

Reply:
Not sure I understand.

Reply:
Nevermind, I do. It is a stupid error. I will fix it.

Reply:
@borgmanJeremy Done.

Reply:
Something weird happened.... I see you pushed 2 commits:

3df20d3: Fixed issue
d2a83c8: Reintroduced issue

![image](https://user-images.githubusercontent.com/46930769/128604639-48ebf27f-d729-498f-a1ea-f8581c9f647c.png)

Reply:
No, this was intentional. Now the only time it doesn't get executed is in debug builds on linux. I just translated the comment into a preprocessor directive. You don't want this?

Reply:
Ohhhh I see the change now. I'll do some testing and then merge it. I want to test against a bunch of DEs